services/121-service/src/financial-service-providers/financial-service-providers.const.ts

```diff
@@ -27,6 +27,7 @@ export const FINANCIAL_SERVICE_PROVIDERS: FinancialServiceProviderDto[] = [
     ],
   },
   {
+    // @ts-expect-error XXX: figure out how to handle this
```

Comment:
FYI @RubenGeo you will probably have to fix this on the feature branch before merging to main. Lmk if it's an issue